crates/ruff/src/rules/flake8_commas/snapshots/ruff__rules__flake8_commas__tests__COM81.py.snap

```diff
@@ -548,7 +548,7 @@ COM81.py:498:13: COM819 [*] Trailing comma prohibited
 496 496 | foo = ['a', 'b', 'c', ]
 497 497 | 
 498     |-bar = { a: b, }
-    498 |+bar = { a: b }
+    498 |+bar = { a: b}
```

Comment:
Hmm, this strikes me as an incorrect fix though. We should probably (?) be mirroring the spacing around the opening bracket, though I'm not sure it's even worth accounting for this in this rule.

Reply:
According to [PEP8 - Pet Peeves](https://peps.python.org/pep-0008/#pet-peeves), this style does not respect PEP8.

Reply:
I agree, I'm just not sure that this rule should be modifying it.